src/remark/variablesPlugin.js

```diff
@@ -1,51 +1,64 @@
-const path = require("path");
-const visit = require("unist-util-visit");
+const path = require("path")
+const visit = require("unist-util-visit")
 
-const variables = require(path.resolve("variables"));
-const objProp = require(path.resolve("src/utils/objProp"));
-
-const pathPrefixes = {
-	cloud: "/docs/calico-cloud/",
-	enterprise: "/docs/calico-enterprise/",
-	openSource: "/docs/calico/",
-};
+const variables = require(path.resolve("variables"))
+const objProp = require(path.resolve("src/utils/objProp"))
+const varRegex = RegExp(/\{\{[ \t]*([\w.]+)[ \t]*}}/, 'g')
 
 function variablesPlugin(_options) {
 	async function transformer(tree, file) {
-		const posixFriendlyPath = convertToPosixFriendlyPath(file.path);
-		const productVariables = posixFriendlyPath.includes(pathPrefixes.cloud)
-			? variables.cloud
-			: posixFriendlyPath.includes(pathPrefixes.enterprise)
-				? variables.enterprise
-				: posixFriendlyPath.includes(pathPrefixes.openSource)
-					? variables.openSource
-					: null;
-
-		if (!productVariables) {
-			return;
-		}
+		const contextVariables = getContextVariables(file, variables)
+		visit(tree, () => true, (node) => {
```

Comment:
If I'm not mistaken, we can just omit the second parameter to allow all types of nodes to be processed.

Reply:
Good to know, thanks. I think we'll just keep it as-is since I know it works :-) - and it's less of a mystery as to what's going on.